packages/ddex/processor/src/publishRelease.ts

```diff
@@ -160,6 +160,8 @@ export async function publishRelease(
   }
 }
 
+// TODO: remove or make use of this
```

Comment:
@stereosteve looks like you wrote this. Is `updateTrack` waiting to be used soon, or can we remove it?